javascript/0125-valid-palindrome.js

```diff
@@ -55,3 +55,31 @@ var isPalindrome = function(s) {
     }
     return true;
 };
+
+/**
+ * 2 Pointer | Midde Convergence | No RegEx | No Copying
+ * Time O(N) | Space O(1)
+ * https://leetcode.com/problems/valid-palindrome/
+ * @param {string} s
+ * @return {boolean}
+ */
+var isPalindrome = function (s) {
+  const isAlpha = c => (c.toLowerCase() >= 'a' && c.toLowerCase() <= 'z') || c >= '0' && c <= '9'
+
+  let i = 0;
```

Comment:
Suggestion: use a more readable variable name like 'left' and 'right'

Reply:
I did some refactoring per your suggestion